pyhip/hip.py

```diff
@@ -1709,3 +1690,27 @@ def hipRuntimeGetVersion():
         ctypes.byref(c_version))
     hipCheckStatus(status)
     return c_version.value
+
+if hipDriverGetVersion() < 50200000:
```

Comment:
I think we can remove this.

Even though the API might not have been implemented, the shared lib should have the definition

Reply:
Do you mean remove the if statement? Because without it I get an error when running PyHIP with newer versions of HIP because hipDeviceSetLimit is not defined.